components/script/dom/bluetooth.rs

```diff
@@ -33,7 +38,48 @@ impl BluetoothMethods for Bluetooth {
 
     // https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetooth-requestdevice
     fn RequestDevice(&self) -> Option<Root<BluetoothDevice>> {
-        //UNIMPLEMENTED
-        None
+        let (sender, receiver) = ipc::channel().unwrap();
+        self.global().r().as_window().bluetooth_thread().send(BluetoothMethodMsg::RequestDevice(sender)).unwrap();
```

Comment:
If we want change this code in the future, maybe write FIXME or TODO above this line and the other occurences  ?